### PR TITLE
add github action for tests + fix dependencies

### PR DESCRIPTION
**Pull request by wpoely86**

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 Looks like the tests are not as expected yet in GitHub Actions?
Cfr. https://github.com/wpoely86/vsc-install/actions

**Comment by wpoely86** [comment]
You mean: your jenkins has some magic combo where it does work :wink: 

Anyway, I've almost got it.

**Comment by wpoely86** [comment]
@boegel fix now: https://github.com/wpoely86/vsc-install/actions/runs/3559909081

File: .github/workflows/unittest.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: run python tests
+on: [push, pull_request]
+
+jobs:
+  python_unittests:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python: [2.7, 3.6, 3.7, 3.8, 3.9]
```
Comment (wdpypere) [inline .github/workflows/unittest.yml:9]: I would only test for 2.7, 3.6 and 3.9 as rhel8 uses 3.6 and rhel9 uses 3.9. But it also should not make that much difference.

Comment (wpoely86) [reply .github/workflows/unittest.yml]: true, I've trimmed it to 2.7, 3.6, 3.9 and 3.11.

**Review by boegel — APPROVED** [review]
lgtm

**Pull request merged**